src/com/clearfaun/mdlag/Sheep.java

```diff
@@ -9,12 +9,13 @@
     //
```

Comment:
Don't do this. It is very annoying for reading. Leave space after the forward slash like this: // Concrete prototype.